content/applications/finance/payment_acquirers/razorpay.rst

```diff
@@ -0,0 +1,44 @@
+===========
+Razorpay
+===========
+
+`Razorpay <https://razorpay.com/>`_ is an online payments provider established in India and
+covering 100+ payment methods.
+
+.. _payment_acquirers/razorpay/configure_dashboard:
+
+Configuration on Razorpay Dashboard
+======================================
+
+#. Log into `Razorpay Dashboard <https://dashboard.razorpay.com/>`_ and go to
+   :menuselection:`Settings --> API Keys`. Generate the new keys and copy the values of the
+   :guilabel:`Key Id` and :guilabel:`Secret Key` fields and save them for later.
+#. | Go to :menuselection:`Settings --> Webhooks` and enter your Odoo database URL followed by
+     `/payment/razorpay/webhook` in the :guilabel:`Webhook URL` text field.
+   | For example: `https://yourcompany.odoo.com/payment/razorpay/webhook`.
+#. Fill the :guilabel:`Secret` field with a password of your choice and save it for later.
+#. Make sure  :guilabel:`refund.processed` and :guilabel:`payment.captured` checkboxes are ticked.
+#. Click on **Save** to finalize the configuration.
```

Comment:
This one slipped my attention, it should be :guilabel:`Save` since this a GUI element. My bad! :)

Reply:
@toaa-odoo , Done..! Thanks